source/docs/casper/resources/tutorials/advanced/cross-contract.md

```diff
@@ -0,0 +1,538 @@
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+# Cross Contract Communication
+
+Level of the tutorial: `Advanced`
+
+Time to complete the tutorial: `45 - 60 Minutes`
+
+Important topics to read before going further:
+
```

Comment:
"this tutorial assumes that you have worked through these examples, if you have not already done so then we recommend that you do so now". Make. Things. CLEAR.

Reply:
This was changed according to the suggestion